meta.yml

```diff
@@ -55,7 +55,7 @@ supported_ocaml_versions:
   opam: '{>= "4.05.0"}'
 
 tested_coq_nix_versions:
-- version: v8.14
+- coq_version: v8.14
```

Comment:
It's a bit confusing that the variable is called `coq_version`, but then the CI configuration uses `coq = <coq_version>`. Why not just call it `version` like for opam/Docker?

Reply:
OK, I guess it makes sense in that context. It's regrettable that we seemingly can't handle opam and Nix dependencies in one go, but it's fine by me to figure this out at some later point.

Reply:
I think we will never be able to completely factorize things because the version selection mechanisms are just too different (Nix has no constraint resolution mechanism).